llama_index/evaluation/base.py

```diff
@@ -31,6 +31,12 @@ class EvaluationResult(BaseModel):
             " presented answers or flipped order"
         ),
     )
+    invalid_result: bool = Field(
+        default=False, description="Whether the evaluation result is an invalid one."
+    )
+    invalid_reason: Optional[str] = Field(
+        default=None, description="Reason for invalid evaluation."
+    )
```

Comment:
This is so that we can not have errors in parsing or others break a batch eval job. We just mark the eval result as invalid and continue along with rest of batch.